cumulus_library/study_parser.py

```diff
@@ -216,6 +258,12 @@ def clean_study(
                 progress,
                 task,
             )
+        if stats_clean:
+            drop_query = get_drop_view_table(
+                f"{drop_prefix}{ProtectedTables.STATISTICS.value}", "TABLE"
+            )
+            cursor.execute(drop_query)
```

Comment:
Why this block? Wouldn't the table normally be picked up by prefix matching already? (the block that strips the table out of the normal list above doesn't strip it if `stats_clean` is on)

Reply:
This is the table that contains the list of protected tables - I guess it could also list itself? but i'll add a comment describing the existing behavior.